flask_security/views.py

```diff
@@ -45,6 +45,8 @@ def _render_json(form, include_user=True, include_auth_token=False):
         response = dict()
         if include_user:
             response['user'] = dict(id=str(form.user.id))
```

Comment:
What about moving this code to default implementation in `flask_security.code:UserMixin.get_security_payload()`.

```python
class UserMixin(BaseUserMixin):
    # ...
    def get_security_payload(self):
        """Serialize user object as response payload."""
        return {'id': str(self.id)}
```

```diff
-             response['user'] = dict(id=str(form.user.id))
+            response['user'] = form.user.get_security_payload()
```

Please make sure that you include tests and documentation for this new feature.